bot/cogs/snakes.py

```diff
@@ -16,32 +19,88 @@ def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
     async def get_snek(self, name: str = None) -> Dict[str, Any]:
-        """
-        Go online and fetch information about a snake
+        with open('bot/db/snakes.json', 'r') as file:
+                snakes_dict = json.load(file)
 
-        The information includes the name of the snake, a picture of the snake, and various other pieces of info.
-        What information you get for the snake is up to you. Be creative!
+        if name is None or len(name) == 0:
+            _, snake_info = random.choice(list(snakes_dict.items()))
 
-        If "python" is given as the snake name, you should return information about the programming language, but with
-        all the information you'd provide for a real snake. Try to have some fun with this!
+        elif len(name) > 0:
+            snake = snakes_dict[name.lower()]
```

Comment:
This will break if the snake isn't in the dict.

Reply:
How can I send an error message to the discord chat if this is just a normal function?

Reply:
You can't.

Reply:
Maybe look into exception handling?

Reply:
So should it just not prompt the user if the snake cannot be found?

Reply:
If it doesn't exist, could I return `snake` as a string and test if it is a string in `get()` and just send a message if it is.

Reply:
You might prefer to return `None` or something, but yes

Reply:
Well, the approach is up to you. Do what works for you.